{"uuid":"a18941e5-2792-4ca9-ab90-fa3d55f90a2a","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["e8bcae6b-6bf4-4f7b-b2b6-592ddec166b4"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592391726,"stop":1681592391727},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592392336,"stop":1681592392337},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592392473,"stop":1681592392473},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592392552,"stop":1681592392553},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681592392576,"stop":1681592392576}],"start":1681592388979,"stop":1681592392576}